.circleci/config.yml

```diff
@@ -140,11 +140,10 @@ jobs:
                   git checkout -b "$branch"
                   git push origin "$branch"
 
-                  # In case of a PR build, CI_PULL_REQUEST should be a link of the form https://github.com/VirtusLab/git-machete-intellij-plugin/pull/123
```

Comment:
This code is only executed for `master` builds... hence `CI_PULL_REQUEST` in undefined